src/horust/healthcheck.rs

```diff
@@ -252,7 +248,7 @@ file-path = "{}""#,
         thread::spawn(move || {
             start_server();
         });
-        assert_eq!(client_mock(), true);
+        assert_eq!(check_http_endpoint("http://127.0.0.1:9999"), true);
```

Comment:
no need to assert_eq, `cargo clippy` should suggest it I think.
```suggestion
        assert!(check_http_endpoint("http://127.0.0.1:9999"));
```

Reply:
Also, you should bind a random port.